### ISSUE TITLE
Error thrown when trying to create new recording schedule in tv/detail/

### ISSUE THREAD

**Issue by mymatenige** [MythTV/mythweb#58]
If I search for a program, find it, look at the detail and set it to record, the following error is thrown:

```
  error type:  Warning
error string:  !!NoTrans: Use of undefined constant Math - assumed 'Math' (this will throw an Error in a future version of PHP)!!
    filename:  modules/tv/classes/Schedule.php
  error line:  295
```

**Comment by mymatenige** [comment]
@bennettpeter Any chance that this can also be merged into master for PHP 8.0?

**Comment by bennettpeter** [comment]
> @bennettpeter Any chance that this can also be merged into master for PHP 8.0?

Sure - I will give it a quick test and if all is well I will merge it.

**Comment by bennettpeter** [comment]
Fixed in 38bee4a2 with pull request #59

**Comment by mymatenige** [comment]
Thanks Peter.
